.eleventy.js

```diff
@@ -11,6 +13,10 @@ module.exports = function(config) {
 
   config.addWatchTarget("./src/styles/");
 
+  config.addPairedNunjucksShortcode("markdown", (content) => {
+    return new markdownIt({ html: true }).render(content);
+  });
```

Comment:
I'm not sure why this is necessary but I suspect that the render sequencing is a bit funky when rendering out data from a json file.